hack/scripts/direct_lvm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# WARNING: THIS SCRIPT HAS MUTLIPLE PURPOSES.
+# TAKE CARE WHEN EDITING.
+
+# This is a scripted version of https://docs.docker.com/storage/storagedriver/device-mapper-driver/#configure-direct-lvm-mode-manually
+
+set -ex
+
+if [[ $(id -u) != 0 ]]; then
+  echo "Run this script as root..." >&2
+  exit 1
+fi
+
+DISK_NAME="${1:-sdb}"
```

Comment:
I think if disk name was not defined, we should exit, that can lead to very serious issues if called without an argument, I'm not sure I would be happy if someone executed this script on my server where `sdb` is NOT empty.

Reply:
yeh good point.

i did it like this to be easy with the bootstrap.sh, as passing a flag there would be annoying 🤔

Reply:
Looks like this good after removing the default disk name

Reply:
👍  got distracted doing other things for a while 😄 

pr comment changes coming up soon